src/persist-txn/src/operator.rs

```diff
@@ -94,7 +94,6 @@ pub fn txns_progress<K, V, T, D, P, C, F, G>(
     txns_id: ShardId,
     data_id: ShardId,
     as_of: T,
-    // TODO(txn): Get rid of these when we have a schemaless WriteHandle.
```

Comment:
I've investigated schemaless WriteHandle and it's a bit more tricky than I'd like. We have real schemas here, may as well use them